api/pull.go

```diff
@@ -589,6 +589,14 @@ func pullHostStatus(host Node, respChan chan<- *httpResponse, dt *Dt, wg *sync.W
 	// use fake interface implementation for tests
 	timeout := time.Duration(dt.Cfg.FlagPullTimeoutSec) * time.Second
 	body, statusCode, err := dt.DtDCOSTools.Get(url, timeout)
+	if statusCode != http.StatusOK {
+		logrus.Errorf("Bad response code %d. URL %s", statusCode, url)
+		response.Status = statusCode
+		host.Health = 3
+		response.Node = host
+		respChan <- &response
```

Comment:
Here you send a pointer to the response after you set response.Node=host.  Below you set response.Node = host after you send it.  does order matter here?

Reply:
no, it does not matter